frontend/appflowy_flutter/lib/plugins/ai_chat/presentation/chat_input/mobile_ai_prompt_input.dart

```diff
@@ -291,3 +297,97 @@ class _MobileAIPromptInputState extends State<MobileAIPromptInput> {
     );
   }
 }
+
+class _LeadingActions extends StatefulWidget {
+  const _LeadingActions({
+    required this.chatId,
+    required this.textController,
+    // required this.onMention,
+    required this.onUpdateSelectedSources,
+  });
+
+  final String chatId;
+  final TextEditingController textController;
+  // final void Function() onMention;
+  final void Function(List<String>) onUpdateSelectedSources;
+
+  @override
+  State<_LeadingActions> createState() => _LeadingActionsState();
+}
+
+class _LeadingActionsState extends State<_LeadingActions> {
+  // bool inputNotEmpty = false;
+  // bool userExpandOverride = false;
+
+  // @override
+  // void initState() {
+  //   super.initState();
+  //   widget.textController.addListener(onTextChanged);
+  // }
+
+  // @override
+  // void dispose() {
+  //   widget.textController.removeListener(onTextChanged);
+  //   super.dispose();
+  // }
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      color: Theme.of(context).cardColor,
+      child: PromptInputMobileSelectSourcesButton(
+        chatId: widget.chatId,
+        onUpdateSelectedSources: widget.onUpdateSelectedSources,
+      ),
+    );
+
+    // return !inputNotEmpty || userExpandOverride
```

Comment:
there are some comment out code, can we remove them?

Reply:
This is the mention page feature. Can I keep it so that I don't have to implement this again in the future?